platform/osx/os_osx.mm

```diff
@@ -504,7 +504,7 @@ - (void)mouseEntered:(NSEvent *)event {
 
 	if (OS_OSX::singleton->input)
 		OS_OSX::singleton->cursor_shape = OS::CURSOR_MAX;
-		OS_OSX::singleton->set_cursor_shape(OS::CURSOR_ARROW);
+	OS_OSX::singleton->set_cursor_shape(OS::CURSOR_ARROW);
```

Comment:
Looks like there's a bug here (should probably be in the `if`).

Reply:
@akien-mga I've made the correction.